ui/src/layouts/chats/presentation/quick_profile/mod.rs

```diff
@@ -413,6 +429,17 @@ pub fn QuickProfileContext<'a>(cx: Scope<'a, QuickProfileProps<'a>>) -> Element<
             }
         ))
         ,
+        share_did.is_some().then(||{
```

Comment:
No need to change, but you could possibly use `map` and add the components there so if `share_did` is `Some(_)`, it can be used, otherwise it will be ignored if its `None`